wapitiCore/language_sources/fr.po

```diff
@@ -467,6 +467,10 @@ msgstr "Injection SQL Sybase"
 msgid ".NET SQL Injection"
 msgstr "Injection SQL .NET"
 
+#: ../language/vulnerability.py:76
+msgid "Basic Authentication Brute Force"
+msgstr "Basique Brute-force Authentification"
+
```

Comment:
Translations for "Basic Authentication Brute Force description" and "Basic Authentication Brute Force solution" are missing.

Reply:
fixed